backend/Cargo.toml

```
@@ -22,11 +22,11 @@
   tracing = { version = "0.1" }
   tracing-subscriber = { version = "0.3.18", features = [ "env-filter" ] }
 
-  strata-bridge-primitives = { git = "https://github.com/alpenlabs/strata-bridge.git" }
+  strata-bridge-primitives = { git = "https://github.com/alpenlabs/strata-bridge.git", tag = "v0.2.0-rc2" }
   strata-bridge-rpc = { git = "https://github.com/alpenlabs/strata-bridge.git", features = [
     "client",
-  ] }
-  strata-primitives = { git = "https://github.com/alpenlabs/alpen.git" }
+  ], tag = "v0.2.0-rc2" }
+  strata-primitives = { git = "https://github.com/alpenlabs/alpen.git", tag = "v0.2.0-rc4" }
```

```suggestion
  strata-primitives = { git = "https://github.com/alpenlabs/alpen.git", tag = "v0.2.0-rc6" }
```
tagged today as well

---

@storopoli : there are mismatches in Buf32 definitions between the two versions `alpen v0.2.0-rc6` and `strata-bridge v0.2.0-rc3`.
```
two different versions of crate `strata_primitives` are being used; two types coming from two different versions of the same crate are different types even if they look the same
   --> /Users/drpaudel/.cargo/git/checkouts/alpen-c4210c7eb647f715/e136896/crates/primitives/src/buf.rs:67:1
    |
67  | pub struct Buf32(pub [u8; 32]);
    | ^^^^^^^^^^^^^^^^ this is the expected type `Buf32`
    |
   ::: /Users/drpaudel/.cargo/git/checkouts/alpen-c4210c7eb647f715/dd4f4d7/crates/primitives/src/buf.rs:67:1
    |
67  | pub struct Buf32(pub [u8; 32]);
    | ^^^^^^^^^^^^^^^^ this is the found type `strata_primitives::buf::Buf32`
```